aptos-move/framework/supra-framework/sources/container.move

```diff
@@ -0,0 +1,82 @@
+module supra_framework::container {
+
+    use std::vector;
+    use aptos_std::simple_map;
+    use aptos_std::simple_map::SimpleMap;
+    use aptos_std::table::Table;
+    use supra_framework::system_addresses;
+
+    struct AddressToContainerMap has key, store {
+        address_container_map: SimpleMap<address, vector<u64>>,
```

Comment:
Not sure why this is a map from `address` to `vector<u64>`, it is supposed to be from module address to container id. Assuming container id to be a unique sequence number, this can be a map `address -> u64`, or rather it can be `Table<address,u64>`.